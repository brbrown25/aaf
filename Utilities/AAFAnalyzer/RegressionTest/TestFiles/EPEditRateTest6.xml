<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE aaf-file SYSTEM "EPDerivationChain.dtd">

<aaf-file>

    <!-- Test one constant edit rate and a different constant sample rate -->

    <clip name="Clip 1">
        <source-clip slot-type="timeline" edit-rate-denominator="24" edit-rate-numerator="600" track-type="audio">
            <file-source name="File Source 1" sample-rate-denominator="24" sample-rate-numerator="1200000">
                <eoc/>
            </file-source>
        </source-clip>
    </clip>
    <clip name="Clip 2">
        <source-clip slot-type="timeline" edit-rate-denominator="24" edit-rate-numerator="600" track-type="audio">
            <file-source name="File Source 1" sample-rate-denominator="24" sample-rate-numerator="1200000">
                <eoc/>
            </file-source>
        </source-clip>
    </clip>

</aaf-file>
