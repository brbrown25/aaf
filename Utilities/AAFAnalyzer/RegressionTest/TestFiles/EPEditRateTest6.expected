        Name:   Edit Protocol Edit Rate Test
        Desc:   Verify that all essence tracks have valid edit rates.
        Result: Passed
        Passing Requirements: REQ_EP_091; REQ_EP_092; REQ_EP_099; REQ_EP_100
        Warning Requirements: None
        Failing Requirements: None

